qubes-rpc/policies/90-default-video-companion.policy

```diff
@@ -0,0 +1,15 @@
+## Do not modify this file, create a new policy file with a lower number in the
+## filename instead. For example `30-user.policy`.
+
+###
+### Policy Qubes Video Companion policy
+###
+
+qvc.ScreenShare * @anyvm @adminvm ask
+qvc.ScreenShare * @anyvm @default ask
+qvc.ScreenShare * @anyvm @anyvm   ask
+qvc.ScreenShare * @anyvm @dispvm  allow
```

Comment:
```suggestion
qvc.ScreenShare * @anyvm @dispvm allow
qvc.ScreenShare * @anyvm * ask
```

`*` matches _anything_, including `@adminvm`.

Reply:
It seems to work this, way indeed, but do we have it documented as inteded behavior anywhere?

Anyway it may be better to to include explicit rule for dom0 anyway so it's clear it's intentional.
But also, I don't the rule for `@adminvm` in the actual content pushed, outside of @DemiMarie 's comment?

Reply:
Ah, it was outdated github UI

Reply:
Added `@adminvm` explictly.